modules/core/src/main/scala/qq/droste/fixie.scala

```diff
@@ -0,0 +1,189 @@
+package qq.droste
+
+import scala.reflect.macros.blackbox
+import scala.language.experimental.macros
+import scala.annotation.StaticAnnotation
+import scala.annotation.compileTimeOnly
+
+@compileTimeOnly("enable macro paradise to expand macro annotations")
+class fixie extends StaticAnnotation {
```

Comment:
How do you feel about renaming `fixie` to `deriveFixedPoint`?

Reply:
Sure thing! Now that it'll be part of droste, the old name doesn't make sense